x/tls_helper.go

```diff
@@ -55,9 +55,55 @@ func RegisterClientTLSFlags(flag *pflag.FlagSet) {
 		"provided by the client to the server.")
 }
 
+func RegisterDgraphTLSFlags(flag *pflag.FlagSet) {
+	flag.String("dgraph_tls_dir", "",
+		"Path to directory that has mTLS certificates and keys for dgraph internal communication")
+	flag.String("dgraph_tls_client_name", "",
+		"client name to be used for mTLS for dgraph internal communication")
+	flag.String("dgraph_tls_server_name", "",
```

Comment:
we dont need to differentiate client and server names. They can be the same thing.